git-remote-gosh/src/git_helper/mod.rs

```diff
@@ -251,6 +260,7 @@ pub async fn run(config: Config, url: &str) -> anyhow::Result<()> {
             (Some("capabilities"), None, None) => helper.capabilities().await?,
             (Some("list"), None, None) => helper.list(false).await?,
             (Some("list"), Some("for-push"), None) => helper.list(true).await?,
+            (Some("repo_version"), None, None) => helper.print_repo_version().await?,
```

Comment:
I think it's better to name gosh related commands with "gosh_" prefix

Reply:
Fixed